src/structures/Invite.js

```diff
@@ -91,6 +92,17 @@ class Invite extends Base {
     return this.createdTimestamp ? new Date(this.createdTimestamp) : null;
   }
 
+  /**
+   * Whether the invite is deletable by the client user
+   * @type {boolean}
+   * @readonly
+   */
+  get deletable() {
+    const guild = this.guild || this.channel.guild;
```

Comment:
Is there any case where `this.guild` may be `null` but `this.channel.guild` is not?